.github/workflows/build.yml

```diff
@@ -85,7 +89,7 @@ jobs:
           rpmsign --addsign \
             --define='%_gpg_name ${{ env.GPG_NAME }}' \
             --define='%_signature gpg' \
-            $(find ./${{ env.REPO_PATH }}/packages -name *.rpm -type f)
+            "$(find ./${{ env.REPO_PATH }}/packages -name '*.rpm' -type f)"
```

Comment:
I'm assuming this is because shellcheck barked? But, I've encountered places where satisfying shellcheck breaks things like this (it treats the entire return from the subshell as a single, space- and/or newline-delimited string rather than the list-type content the non-quoted subshell would return). Is this verified to work as expected?

Reply:
Also, within the context of a subshell, you can still use doublequotes even if you put doublequotes around the `()`: the subshell-invoking parens protect the interior doublequotes from the higher-level shell(s).

Reply:
I don't have all the packages built locally at the moment, but the general syntax appears to be doing the right thing:

```
❯ echo "$(find . -name '*.yml' -type f)"
./.github/dependabot.yml
./.github/workflows/build.yml
./.github/workflows/release.yml
./.github/workflows/review.yml
```

Reply:
Kul. Like I say, I've encountered use-cases where I just had to tell shellcheck to pound sand. So, wanted to raise it as a possible problem.